pkg/storage/export/virt-exportserver/exportserver.go

```diff
@@ -196,6 +197,9 @@ func (s *exportServer) Run() {
 	srv := &http.Server{
 		Addr:    s.ListenAddr,
 		Handler: s.handler,
+		// Disable HTTP/2
+		// See CVE-2023-44487
+		TLSNextProto: map[string]func(*http.Server, *tls.Conn, http.Handler){},
```

Comment:
This is already a case per my understanding, did you find anything else?

Reply:
It is my understanding that HTTP/2 is enabled by default and I've seen many PRs disabling it in this fashion.  The doc for this field follows.  It's confusing but see:  "If TLSNextProto is not nil, HTTP/2 support is not enabled automatically."  Meaning support is enabled if nil

// TLSNextProto specifies how the Transport switches to an
	// alternate protocol (such as HTTP/2) after a TLS ALPN
	// protocol negotiation. If Transport dials an TLS connection
	// with a non-empty protocol name and TLSNextProto contains a
	// map entry for that key (such as "h2"), then the func is
	// called with the request's authority (such as "example.com"
	// or "example.com:1234") and the TLS connection. The function
	// must return a RoundTripper that then handles the request.
	// If TLSNextProto is not nil, HTTP/2 support is not enabled
	// automatically.